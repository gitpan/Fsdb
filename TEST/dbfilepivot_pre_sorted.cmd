prog='dbfilepivot'
args='-S -k name -p hw -v score'
cmp='diff -cb '
in=TEST/dbfilepivot_ex.in
