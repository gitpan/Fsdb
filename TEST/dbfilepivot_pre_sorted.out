#fsdb name hw_1 hw_2 hw_3
John	97	98	99
Paul	-	80	82
#   | dbfilepivot -S -k name -p hw -v score
